packages/search-ui-workplace-search-connector/src/WorkplaceSearchAPIConnector.ts

```diff
@@ -177,24 +175,50 @@ class WorkplaceSearchAPIConnector {
     }
   }
 
-  onResultClick({
-    query,
-    documentId,
-    requestId,
-    tags = []
-  }: ResultClickParams): void {
-    tags = tags.concat("results");
-    return this.client.click({ query, documentId, requestId, tags });
+  onResultClick({ documentId, requestId }: ResultClickParams): void {
+    const apiUrl = `${this.enterpriseSearchBase}/api/ws/v1/analytics/event`;
+
+    this.performFetchRequest(apiUrl, {
+      type: "click",
+      document_id: documentId,
+      query_id: requestId,
+      content_source_id: null, // todo: where would this be available?
+      page: 1 // todo: where can i reach to meta?
+    });
   }
 
   onAutocompleteResultClick({
-    query,
     documentId,
-    requestId,
-    tags = []
+    requestId
   }: ResultClickParams): void {
-    tags = tags.concat("autocomplete");
-    return this.client.click({ query, documentId, requestId, tags });
+    const apiUrl = `${this.enterpriseSearchBase}/api/ws/v1/analytics/event`;
+
+    this.performFetchRequest(apiUrl, {
+      type: "click",
+      document_id: documentId,
+      query_id: requestId,
+      content_source_id: null, // where would this be available?
```

Comment:
yeh this is outstanding and not sure if thats part of the ResultClickParams argument. Need to test this on a real endpoint to see

Reply:
Looking into this further, one route is to extend the `trackClickThrough` on search-ui to provide the raw result to the connector so that we can access the content_source_id thats contained within the meta of each result.